.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -14,6 +14,12 @@ body:
   - type: markdown
     attributes:
       value: 'next@canary is the beta version of Next.js. It includes all features and fixes that are pending to land on the stable release line.'
+  - type: textarea
+    attributes:
+      label: Run `next info`
```

Comment:
We could say "use it from 12.0.8 and up", but that might be confusing. What do you think?

Reply:
Yeah we need to specific the version here and also make it optional